backend/app/auth/security.py

```
@@ -95,3 +98,25 @@
         A random 32-byte URL-safe string suitable for use as a password reset token.
     """
     return secrets.token_urlsafe(32)
+
+def get_current_user(token: str = Depends(oauth2_scheme)) -> Dict[str, Any]:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)